src/components/InputBlock/InputBlock.jsx

```diff
@@ -64,4 +98,5 @@ InputBlock.propTypes = {
   onSubmit: PropTypes.func,
   inputDisabled: PropTypes.bool,
   inputRef: PropTypes.object,
+  pause: PropTypes.bool,
```

Comment:
Maybe it would be better to rename this to something more descriptive like `isPaused`